locales/de/subtitles/vincent-pascucci/vincent-pascucci-01.yml

```diff
@@ -218,11 +211,9 @@ de:
           s: 41
           from: 00:04:17,08
           to: 00:04:30,17
-          text: I worked in a toy factory; I also worked unloading the little boats
-            on the Seine.
+          text: Ich arbeitete in einer Spielzeugfabrik und beim Entladen von Frachtkähnen auf der Seine.
         sequence-42:
           s: 42
           from: 00:04:30,17
           to: 00:04:41,10
-          text: I was not very picky about the work, as it was quite hard to find
-            a job at the time.
+          text: Man nahm die Arbeit, die man haben konnte. Es war schwierig genug Arbeit zu finden.
```

Comment:
I changed it to 

> Man hat sich nicht so angestellt, wenn man Arbeit bekommen konnte

which might be a bit closer to the original:

>  On était pas trop difficile quand on pouvait avoir un travail parce
que c'était assez....